app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,47 @@
+class TasksController < ApplicationController
+  before_action :set_story
+  before_action :set_story_task, only: [:show, :update, :destroy]
+
+  # GET /stories/:story_id/tasks
```

Comment:
I'd move this one to the story controller (to get all tasks for a story), but make the tasks routes independent from stories, since they have a unique ID on their own (I assume?) and hence don't need the nesting

Reply:
I am fine with your suggestion, but I did it  REST-style. Since a task is always part of a story, they should addressed like this. If there can be a task without a story, your point is more valid.

Reply:
both work for REST (imo), since we can address the task uniquely without involving the story. I'm leaving it up to you cause I don't care much, tbh. Just think the easier the urls, the better, as long as they still make sense.